CHANGELOG/CHANGELOG-1.7.md

```diff
@@ -0,0 +1,110 @@
+  * Refer to [this cookbook](https://github.com/antrea-io/antrea/blob/release-1.7/docs/cookbooks/ids/README.md) for more information about using this feature to provide network-based intrusion detection service to your Pods
+- Add support for the IPsec Certificate-based Authentication. ([#3778](https://github.com/antrea-io/antrea/pull/3778), [@xliuxu])
+  * Add an Antrea Agent configuration option `ipsec.authenticationMode` to specify authentication mode. Supported options are "psk" (default) and "cert".
+  * Add an Antrea Controller configuration option `ipsecCSRSigner.autoApprove` to specify the auto-approve policy of Antrea CSR signer for IPsec certificates management. By default, Antrea will auto-approve the CertificateSingingRequest (CSR) if it is verified.
+  * Add an Antrea Controller configuration option `ipsecCSRSigner.selfSignedCA` to specify whether to use auto-generated self-signed CA certificate. By default, Antrea will auto-generate self-signed CA certificate.
+- Add the following capabilities to the AntreaPolicy feature:
+  * Add support for ICMP protocol in Antrea-native policies. ([#3472](https://github.com/antrea-io/antrea/pull/3472), [@GraysonWu])
+  * Add support for IGMP protocol in Antrea-native policies. ([#3660](https://github.com/antrea-io/antrea/pull/3660), [@liu4480])
+  * Add support for matching multicast traffic in Antrea-native policies. ([#3660](https://github.com/antrea-io/antrea/pull/3660), [@liu4480])
+  * Add support for rule-level statistics for IGMP traffic and multicast traffic. ([#3449](https://github.com/antrea-io/antrea/pull/3449), [@ceclinux])
+- Add the following capabilities to the Multicast feature:
+  * Add `antctl get podmulticaststats` command to query Pod-level multicast traffic statistics in Agent mode. ([#3449](https://github.com/antrea-io/antrea/pull/3449), [@ceclinux])
+  * Add "MulticastGroup" API to query Pods that have joined multicast groups; `kubectl get multicastgroups` can generate requests and output responses of the API. ([#3354](https://github.com/antrea-io/antrea/pull/3354) [#3449](https://github.com/antrea-io/antrea/pull/3449), [@ceclinux])
+  * Add an Antrea Agent configuration option `multicast.igmpQueryInterval` to specify the interval at which the antrea-agent sends IGMP queries to Pods. ([#3819](https://github.com/antrea-io/antrea/pull/3819), [@liu4480])
+- Add the following capabilities to the Multicluster feature:
+  * Add a number of `antctl mc` subcommands for bootstrapping Multi-cluster; refer to [antct.md](https://github.com/antrea-io/antrea/blob/release-1.7/docs/multicluster/antctl.md) for more information. ([#3474](https://github.com/antrea-io/antrea/pull/3474), [@hjiajing])
+  * <Placeholder for Gateway, ClusterInfo related changes>. ([#3689](https://github.com/antrea-io/antrea/pull/3689) [#3463](https://github.com/antrea-io/antrea/pull/3463) [#3603](https://github.com/antrea-io/antrea/pull/3603), [@luolanzone])
```

Comment:
@luolanzone could you help summary the changes we made in 1.7 for Gateway and ClusterInfo?

Reply:
Hi, @tnqn here is the info:

 Add Gateway and ClusterInfoImport CRDs and controllers to allow basic cluster information (Service CIDR, Gateway IPs etc.) exchange between member clusters. ([#3689](https://github.com/antrea-io/antrea/pull/3689) [#3463](https://github.com/antrea-io/antrea/pull/3463), [@luolanzone])
 Add Multicluster feature to support direct Service access cross clusters through Gateway Node. [#3603](https://github.com/antrea-io/antrea/pull/3603), [@luolanzone])

Reply:
thanks, added